_download-meta/openttd-nightlies.md

```diff
@@ -2,9 +2,13 @@
 folder_old_infrastructure: nightlies/trunk
 ---
 
-For OpenTTD you can use the original Transport Tycoon Deluxe data files (you need to own a Transport Tycoon Deluxe CD).
-There are also the free alternatives: [OpenGFX (graphics)](http://dev.openttdcoop.org/projects/opengfx), [OpenSFX (sound)](http://dev.openttdcoop.org/projects/opensfx), and [OpenMSX (music)](http://dev.openttdcoop.org/projects/openmsx).
-These can be installed automatically by the Windows and OS/2 installers.
-Please refer to the [readme](https://proxy.binaries.openttd.org/openttd-nightlies/@@version@@/README.md) for more information.
 
-You can download the free alternatives here: [download OpenGFX](../opengfx-releases/latest.html), [download OpenSFX](../opensfx-releases/latest.html), and [download OpenMSX](../openmsx-releases/latest.html).
+OpenTTD requires one version of base graphics. Available for installation with the installer are:
+* [OpenGFX (graphics)](http://dev.openttdcoop.org/projects/opengfx)
+* Transport Tycoon Deluxe data files can be installed if you own the Transport Tycoon Deluxe CD
+
+Sound and music sets are recommended (but not necessary). [OpenSFX (sound)](http://dev.openttdcoop.org/projects/opensfx), and [OpenMSX (music)](http://dev.openttdcoop.org/projects/openmsx) can be installed via the installer or - if you own the CD -  also the Transport Tycoon Deluxe sound and music.
```

Comment:
Please put every sentence on its own line (so \n after dot). This makes markdown diffs a lot easier to read in the future :)